tlspuffin/src/tls/key_schedule.rs

```diff
@@ -75,14 +69,18 @@ pub fn tls13_derive_psk(
     server_key_share: &Option<Vec<u8>>,
     new_ticket_nonce: &[u8],
     group: &NamedGroup,
+    client_random: &Random,
+    suite: &SupportedCipherSuite,
 ) -> Result<Vec<u8>, FnError> {
-    let (_, _, pending) = tls13_application_traffic_secret(
+    let (_, pending) = tls13_application_traffic_secret(
         server_hello,
         server_finished,
         server_key_share,
         &None,
```

Comment:
No PSK? I guess other functions is in charge of PSK. Please double check.

Reply:
This is the method to derive the psk from the session